### PR TITLE
Add some details about getting started with Vim.

### PR DESCRIPTION
**Pull request by artasparks**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +32,25 @@ If you're new to Git / Github, check out:
         Specifically, we use an [Integration-Manager workflow (Github)](https://git-scm.com/book/en/v2/Distributed-Git-Distributed-Workflows#wfdiag_b).
 2.  [Getting Started with Github](https://help.github.com/en/github/getting-started-with-github)
 
+## Editors
+
+You are, of course, free to use whichever text editor you feel most comfortable
+with. Many people working on Clamshell use Vim or IntelliJ, so we can help you
```
Comment (artasparks) [inline CONTRIBUTING.md:38]: Emacs is Better than Vim!

Comment (artasparks) [reply CONTRIBUTING.md:38]: Nah bro.

Comment (RodDeacon) [reply CONTRIBUTING.md:38]: nice meme

**Review by RodDeacon — APPROVED** [review]
Thanks, looks good.. 

Approved

**Pull request merged**